crates/bevy_pbr/src/light.rs

```diff
@@ -20,6 +20,61 @@ use bevy_utils::{tracing::warn, EntityHashMap};
 
 use crate::*;
 
+/// Constants for operating with the light units: lumens, and lux.
+pub mod light_consts {
+    /// Approximations for converting the wattage of lamps to lumens.
+    ///
+    /// The **lumen** (symbol: **lm**) is the unit of [luminous flux], a measure
+    /// of the total quantity of [visible light] emitted by a source per unit of
+    /// time, in the [International System of Units] (SI).
+    ///
+    /// For more information, see [wikipedia](https://en.wikipedia.org/wiki/Lumen_(unit))
+    ///
+    /// [luminous flux]: https://en.wikipedia.org/wiki/Luminous_flux
+    /// [visible light]: https://en.wikipedia.org/wiki/Visible_light
+    /// [International System of Units]: https://en.wikipedia.org/wiki/International_System_of_Units
+    pub mod lumens {
+        pub const LUMENS_PER_LED_WATTS: f32 = 90.0;
+        pub const LUMENS_PER_INCANDESCENT_WATTS: f32 = 13.8;
+        pub const LUMENS_PER_HALOGEN_WATTS: f32 = 19.8;
+    }
+
+    /// Predefined for lux values in several locations.
+    ///
+    /// The **lux** (symbol: **lx**) is the unit of [illuminance], or [luminous flux] per unit area,
+    /// in the [International System of Units] (SI). It is equal to one lumen per square metre.
+    ///
+    /// For more information, see [wikipedia](https://en.wikipedia.org/wiki/Lux)
+    ///
+    /// [illuminance]: https://en.wikipedia.org/wiki/Illuminance
+    /// [luminous flux]: https://en.wikipedia.org/wiki/Luminous_flux
+    /// [International System of Units]: https://en.wikipedia.org/wiki/International_System_of_Units
+    pub mod lux {
+        /// The amount of light (lux) in a moonless, overcast night sky. (starlight)
+        pub const MOONLESS_NIGHT: f32 = 0.0001;
+        /// The amount of light (lux) in a full moon on a clear night.
+        pub const FULL_MOON_NIGHT: f32 = 0.05;
+        /// The amount of light (lux) in dark limit of civil twilight under a clear sky.
+        pub const CIVIL_TWILIGHT: f32 = 3.4;
```

Comment:
"civil twilight" doesn't make sense in English. Is this trying to capture the twilight of a city perhaps?

Reply:
[It's a legit technical term](https://en.wikipedia.org/wiki/Twilight#Civil_twilight), but not a widely-known one. I'd never heard of it before.

Reply:
I don't think that's needed, it's easy to tell that it's some kind of twilight, which is good enough for most people. And for the people who want to know more, can just look it up.